ocs_ci/utility/version.py

```diff
@@ -132,7 +132,9 @@ def get_ocs_version_from_csv(only_major_minor=False, ignore_pre_release=False):
     csvs = ocp.OCP(
         namespace=config.ENV_DATA["cluster_namespace"], kind="", resource_name="csv"
     )
-    if get_semantic_ocs_version_from_config() >= VERSION_4_9:
+    if config.ENV_DATA["cluster_type"].lower() == constants.HCI_CLIENT:
```

Comment:
version of an ODF client operator is not the same as OCS Operator version. In this case it will be right to switch to Provider and get version of ODF Operator, elsewise we are creating misleading logic here.

 Some reasoning
- if versions of ODF Provider and ODF Client on consumer cluster are different, we will hide a bug. 
- we can not always rely on fact that versions are the matching, this may change in future, or even not be true, for example, during upgrade.
- purpose of this func is to get ocs version from csv of ODF, this must correlate to version of ceph components, and dependency operators like rook-csi, etc.